BoxBrowseSDK/BoxBrowseSDK/Categories/UIImage+BOXBrowseSDKAdditions.m

```diff
@@ -24,31 +24,33 @@ + (UIImage *)box_iconForItem:(BOXItem *)item
         if (item.isFolder) {
             BOXFolder *folder = (BOXFolder *)item;
             if (folder.isExternallyOwned == BOXAPIBooleanYES) {
-                icon = [UIImage imageNamed:@"icon-folder-external"
+                icon = [UIImage imageNamed:@"External"
                                   inBundle:browseResourceBundle
              compatibleWithTraitCollection:nil];
             } else if (folder.hasCollaborations == BOXAPIBooleanYES) {
-                icon = [UIImage imageNamed:@"icon-folder-shared"
+                icon = [UIImage imageNamed:@"shared_folder"
```

Comment:
double checked, it's there, somehow github is overwhelmed by the number of file changed